### DISCUSSION TITLE
Parcel v2 import/require html-files in JavaScript to String

### DISCUSSION THREAD

**Discussion by eirikb** [parcel-bundler/parcel#5798] [Q&A] [answered]
Hi

Is it possible to import `.html` files as strings from JavaScript?  
What I'm trying:
```js
import ok from './ok.html';
console.log(ok);
```
What I get:
![image](https://user-images.githubusercontent.com/241706/107205491-38c55c00-69fe-11eb-97f0-c17d5617fb79.png)

Note: In Parcel v1 I would get the html-file as a string.

**Comment by mischnic** [comment, marked as answer]
```js
import ok from 'bundle-text:./ok.html';
console.log(ok);
```
